homeassistant/components/emulated_hue/upnp.py

```diff
@@ -119,7 +107,7 @@ def run(self):
             socket.inet_aton(self.host_ip_addr))
 
         if self.upnp_bind_multicast:
-            ssdp_socket.bind(("239.255.255.250", 1900))
+            ssdp_socket.bind(("", 1900))
```

Comment:
From your intro:
> Also, the receipt of SSDP messages is via the ANY ip address and not the Multicast address, thus the bind was also incorrect.

Are you sure about this? It works on Linux while being bound to this IP.